deploy/package.sh

```diff
@@ -5,80 +5,103 @@
 
 PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"
 
-ENV_CODE=${1:-${ENV_CODE}}
-PIPELINE_NAME=${2:-${PIPELINE_NAME}}
+ENV_CODE=${1:-$ENV_CODE}
+PIPELINE_NAME=${2:-$PIPELINE_NAME}
 
+AI_MODEL_INFRA_TYPE=${3:-${AI_MODEL_INFRA_TYPE:-"batch-account"}} # Currently supported values are aks and batch-account
 
-BATCH_ACCOUNT_NAME=${3:-${BATCH_ACCOUNT_NAME}}
-BATCH_ACCOUNT_RG_NAME=${4:-$BATCH_ACCOUNT_RG_NAME}
-BATCH_STORAGE_ACCOUNT_NAME=${5:-${BATCH_STORAGE_ACCOUNT_NAME}}
-KEY_VAULT_NAME=${6:-${KEY_VAULT_NAME}}
+AI_MODEL_INFRA_RESOURCE_NAME=${4:-$AI_MODEL_INFRA_RESOURCE_NAME}
+AI_MODEL_INFRA_RG_NAME=${5:-$AI_MODEL_INFRA_RG_NAME}
+AI_MODEL_INFRA_STORAGE_ACCOUNT_NAME=${6:-$AI_MODEL_INFRA_STORAGE_ACCOUNT_NAME}
 
-RAW_STORAGE_ACCOUNT_RG=${7:-${RAW_STORAGE_ACCOUNT_RG:-"${ENV_CODE}-data-rg"}}
-RAW_STORAGE_ACCOUNT_NAME=${8:-${RAW_STORAGE_ACCOUNT_NAME}}
+KEY_VAULT_NAME=${7:-$KEY_VAULT_NAME}
 
-SYNAPSE_WORKSPACE_RG=${9:-${SYNAPSE_WORKSPACE_RG:-"${ENV_CODE}-pipeline-rg"}}
-SYNAPSE_WORKSPACE_NAME=${10:-${SYNAPSE_WORKSPACE_NAME}}
-SYNAPSE_STORAGE_ACCOUNT_NAME=${11:-${SYNAPSE_STORAGE_ACCOUNT_NAME}}
-SYNAPSE_POOL=${12:-${SYNAPSE_POOL}}
+RAW_STORAGE_ACCOUNT_RG=${8:-${RAW_STORAGE_ACCOUNT_RG:-"${ENV_CODE}-data-rg"}}
+RAW_STORAGE_ACCOUNT_NAME=${9:-$RAW_STORAGE_ACCOUNT_NAME}
 
-DEPLOY_PGSQL=${13:-${DEPLOY_PGSQL:-"true"}}
+SYNAPSE_WORKSPACE_RG=${10:-${SYNAPSE_WORKSPACE_RG:-"${ENV_CODE}-pipeline-rg"}}
+SYNAPSE_WORKSPACE_NAME=${11:-$SYNAPSE_WORKSPACE_NAME}
+SYNAPSE_STORAGE_ACCOUNT_NAME=${12:-$SYNAPSE_STORAGE_ACCOUNT_NAME}
+SYNAPSE_POOL=${13:-$SYNAPSE_POOL}
+
+DEPLOY_PGSQL=${14:-${DEPLOY_PGSQL:-"true"}}
 
 MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"
```

Comment:
Now we can use $AI_MODEL_INFRA_TYPE variable in this PR. In the previous, I asked Raj drop this dependency, so this need updating as follows:

MODE="${$AI_MODEL_INFRA_TYPE}$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"

Reply:
updated